src/validate_total_storage_categorized.R

```diff
@@ -16,12 +16,12 @@ validate_total_storage_categorized <- function(data_categorized, n_hrus = 109951
   
   # We know that one CA HRU doesn't have historic data for quantiles (104388) and will be Undefined
```

Comment:
I think you can delete these commented out lines or update this comment to say that in the previous version this was the case but it is no longer an issue now.

Reply:
yeah, i guess you might be seeing the old version of the comment here, strange, you did generalize it in your same commit with the fix for data.table

Reply:
Huh I wonder if that change got overwritten somewhere along the way?

Reply:
ah, this is my mistake, yes I can remove this -- those changes are in the other file 🤦 src/validate_historic_quantile_data.R